### ISSUE TITLE
RSpec/VariableName   gives" has the wrong namespace - should be Naming" when run with --only option

### ISSUE THREAD

**Issue by mhatherly** [rubocop-hq/rubocop-rspec#1073]
rubocop will run the cop unless you use the --only option 

rubocop -f o
 will give this output

> 9    RSpec/VariableName

Indicating 9 viiolations  found, but running 

> rubocop --only RSpec/VariableName

will give 

> --only option: RSpec/VariableName has the wrong namespace - should be Naming
> Inspecting 28 files
> ............................
> 28 files inspected, no offenses detected

version of rubocop-rspec is 1.44.1
version of rubocop is 0.93.1
running in docker container based on Ruby 2.7.2 image from dockerhub

**Comment by pirj** [comment]
I hope we've fixed it with the soon-to-be-released 2.0. Can you please help with testing it? Point your `Gemfile` to `release-2.0`:
```
gem 'rubocop-rspec', github: 'rubocop-hq/rubocop-rspec', branch: 'release-2.0'
```

**Comment by pirj** [comment]
[2.0](https://github.com/rubocop-hq/rubocop-rspec/releases/tag/v2.0.0) has been released, and this issue should be resolved.
Please feel free to reopen if the issue remains.
